### PR TITLE
about.jsp: Add photos for current committers #7046

### LINKED ISSUE #7046: about.jsp: Add photos for current committers
Processed photos are given below. To be added to https://teammatesv4.appspot.com/about.jsp (names are already in that page)
Might need resizing the photos before adding to the repo.

image
Samson

image
Xiao Pu

image
Boxin

image
John

image
Dickson

image
Theanesh

image
Li Kai

image
Ming Shin

image
Yi Min

### PR DESCRIPTION
**Pull request by HirdayGupta**
Fixes #7046 

Resized the images posted in the issue to width 300, and added them to repo.
Changed the Committers section in the about.jsp file to include these images following the design of the Past Team Members section on the same page.

### CODE REVIEW

File: src/main/webapp/about.jsp [hunk 1]
```diff
@@ -153,14 +153,42 @@
...
-                <li>Ch'ng Ming Shin (Jan 2017 - )</li>
-                <li>Dickson Tan (Jan 2017 - )</li>
-                <li>John Yong (Jan 2017 - )</li>
-                <li>Lee Yi Min (Jan 2017 - )</li>
-                <li>Li Kai (Jan 2017 - )</li>
-                <li>Samson Tan (Jan 2017 - )</li>
-                <li>Thenaesh Elango (Jan 2017 - )</li>
-                <li>Xiao Pu (Jan 2017 - )</li>
+                <li>
+                    <img src="images/teammembers/shin.png" width="80px"><br>
```
Comment (whipermr5) [inline src/main/webapp/about.jsp]: `shin.png` -> `mingshin.png`

File: src/main/webapp/about.jsp [hunk 2]
```diff
@@ -153,14 +153,42 @@
...
+                <li>
+                    <img src="images/teammembers/dickson.png" width="80px"><br>
+                    Dickson Tan (Jan 2017 - )
+                </li>
+                <li>
+                    <img src="images/teammembers/john.png" width="80px"><br>
+                    John Yong (Jan 2017 - )
+                </li>
+                <li>
+                    <img src="images/teammembers/min.png" width="80px"><br>
```
Comment (whipermr5) [inline src/main/webapp/about.jsp]: `min.png` -> `yimin.png`

File: src/main/webapp/about.jsp [hunk 3]
```diff
@@ -153,14 +153,42 @@
...
+                <li>
+                    <img src="images/teammembers/samson.png" width="80px"><br>
+                    Samson Tan (Jan 2017 - )
+                </li>
+                <li>
+                    <img src="images/teammembers/thenaesh.png" width="80px"><br>
+                    Thenaesh Elango (Jan 2017 - )
+                </li>
+                <li>
+                    <img src="images/teammembers/pu.png" width="80px"><br>
```
Comment (whipermr5) [inline src/main/webapp/about.jsp]: `pu.png` -> `xiaopu.png`

File: src/main/webapp/about.jsp [hunk 4]
```diff
@@ -153,14 +153,42 @@
...
+                    Xiao Pu (Jan 2017 - )
+                </li>
+                <li>
+                    <img src="images/teammembers/boxin.png" width="80px"><br>
+                    Boxin (Jan 2017 - )
```
Comment (whipermr5) [inline src/main/webapp/about.jsp]: `Boxin` -> `Yang Boxin` (but for the filename, `boxin.png` is fine)

**Comment by HirdayGupta** [comment]
@whipermr5 Completed the requested changes. Here is the screenshot: 

screen shot 2017-04-07 at 11 33 12 pm 

Let me know if any design changes are required.

**Comment by whipermr5** [comment]
@HirdayGupta Looks good! 👍 Remember to update the HTML with the new filenames

**Comment by whipermr5** [comment]
Also, run the images through [Kraken](https://kraken.io/web-interface) to compress them (use the lossless setting).

**Comment by HirdayGupta** [comment]
Done and done. Apologies for the silly HTML oversight. Overall space saved using Kraken was about 2.5%.

**Comment by HirdayGupta** [comment]
@whipermr5 Any idea why Travis says all checks are failing?

**Comment by whipermr5** [comment]
@HirdayGupta Hmm we've been having that error in other PRs too; the JavaScript team is looking into it. Anyway, your latest commit changes the layout? That is beyond the scope of this PR.

**Pull request merged**